backend/controllers/roleDescriptionAmendment.test.js

```diff
@@ -0,0 +1,107 @@
+const { roleDescriptionAmendmentFactory } = require('./roleDescriptionAmendment')
+
+describe('role amendment factory', () => {
+  const getRoleDetailsApi = jest.fn()
+  const changeRoleDescriptionApi = jest.fn()
+  const changeRoleDescription = roleDescriptionAmendmentFactory(
+    getRoleDetailsApi,
+    changeRoleDescriptionApi,
+    'Change role description',
+    '/manage-roles',
+  )
+
+  describe('index', () => {
+    it('should call roleDescription render', async () => {
+      const req = { params: { role: 'role1' }, flash: jest.fn() }
+      getRoleDetailsApi.mockResolvedValue({
+        roleDescription: 'role description',
+      })
+
+      const render = jest.fn()
+      await changeRoleDescription.index(req, { render })
+      expect(render).toBeCalledWith('changeRoleDescription.njk', {
+        currentRoleDescription: 'role description',
+        title: 'Change role description',
+        roleUrl: '/manage-roles/role1',
+        errors: undefined,
+      })
+    })
+
+    it('should copy any flash errors over', async () => {
+      const req = { params: { role: 'role1' }, flash: jest.fn().mockReturnValue({ error: 'some error' }) }
+      getRoleDetailsApi.mockResolvedValue({
+        roleDescription: 'role description',
+      })
+
+      const render = jest.fn()
+      await changeRoleDescription.index(req, { render })
+      expect(render).toBeCalledWith('changeRoleDescription.njk', {
+        errors: { error: 'some error' },
+        currentRoleDescription: 'role description',
+        title: 'Change role description',
+        roleUrl: '/manage-roles/role1',
```

Comment:
very minor, but would it make sense to be 'Change description for <role1>`  ??? rather then the description in title. I suppose we can see what it looks like for real data

Reply:
yes, I will update to be more meaningful in next pr